github/ratelimit.go

```diff
@@ -0,0 +1,219 @@
+package github
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"strconv"
+	"strings"
+	"sync"
+	"time"
+
+	"gopkg.in/src-d/go-log.v1"
+
+	"github.com/src-d/metadata-retrieval/utils"
+)
+
+var (
+	defaultAbuseRetryAfter = time.Minute
```

Comment:
isn't it a lot?

Reply:
> It's the default I saw when they send the Retry-After header, so I just reused it assuming it's what they prefer.

ok. Let's add the comment on why we chose this value then:
```
// this value is usually returned by github
```

Reply:
done by 44e8aa81105656a29e3992e72b64a121f38e55b5